documentation/release_4.1.htm

```diff
@@ -88,8 +90,10 @@ <h3>Changed functionality</h3>
     <li><code>ScatterEstimation</code> has now many more <code>set*</code> functions such that most
       (all?) parameters can be set from C++/Python (i.e. without parsing).
     </li>
+    <li><tt>axpby</tt> has been deprecated and replaced with <tt>xapyb</tt>.</li>      
```

Comment:
can you check, but I'm fairly sure that axpby was introduced by Richard after STIR 4.0. Not 100% sure. (you can check the release notes, or `git blame`). If so, I would rephrase this as "xpaby was introduced on the release_4 branch after STIR 4.0 but is now deprecated...."

Reply:
Yes, it looks like it was introduced in 4.1 so I've made the change.